cmd/build.go

```diff
@@ -5,15 +5,21 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"log"
+	"os"
 	"time"
 
+	"github.com/Azure/acrbuild"
```

Comment:
just copy/paste this in for now in a separate PR so we can unbreak travis and make sure we're both running the same stuff